internal/controller/suite_test.go

```diff
@@ -108,6 +108,15 @@ var _ = BeforeSuite(func() {
 	err = k8sClient.Create(ctx, ns)
 	Expect(err).NotTo(HaveOccurred())
 
+	By("Create the namespace openshift-config")
+	ns = &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "openshift-config",
+		},
+	}
+	err = k8sClient.Create(ctx, ns)
```

Comment:
this NS should exist on every OCP cluster, you should not have to create it.....did you have some issue?

Reply:
ah, ok.  sorry, was lacking context